Day-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,10 @@
 import React from 'react';
 import './Card.css';
+
+const CardBanner = () => {
+  return (
+    <div class ='CardBanner'>ipsum loral</div>
```

Comment:
This component is meant for the card image. What purpose does this div serve in relation to the design file?